QCD shifts and bTagReshape SF normalization

francescobrivio opened the pull request:
Updated `combineFillerOutputs.py` to include:
- QCD shifted: it is built using the "old" method (shape from B and correction factor from C/D), up and down variations are the same.
- New bTagReshape SFs need a rinormalization factor: since it's always the same for all events it can be factorized and applied on the final histogram. Different factors for each different bTagReshape systematic should be used, but until we compute all of them I will use the central value.

Configs
- Updated all `main` and `selection` configs to include 
   - the bTagReshape weights
   - VBF trigger jets legs ucnertainties in tautau 2017 and 2018
- For the moment uncertainties on the bTagReshape SFs are commented out

dzuolo commented on config/selectionCfg_ETau_Legacy2017_syst.cfg:
```diff
@@ -886,15 +886,15 @@ DYclass_jesDown_11           = VBFloose_jesDown_11 , mpp_dy_v2_jesDown_11
 ## NOTE: no weight is applied for data (the simple Fill() is used)
 [selectionWeights]
-baseline  = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT
+baseline  = MC_weight, PUReweight, PUjetID_SF, L1pref_weight, prescaleWeight, trigSF, IdAndIsoAndFakeSF_deep_pt, DYscale_MTT, bTagweightReshape
```
Just a general comment: why are we applying this weight also to the baseline where in the past we did not apply any btagweight?

dzuolo replied:
You're right, I keep forgetting. The QCD part instead seems ok to me.

francescobrivio commented:
Added a fix for the VBF trigger SF in the skimmers.
Instead of re-skimming everything, I added a section (`doVBFtrig`) in the systematic module that creates new fixed branches for the VBF triggers.
@camendola please have a look at commit 24e36f4 to see if it's ok.
